### PR TITLE
Add code of conduct and clean up links

### PR DESCRIPTION
**Pull request by henrykironde**

### CODE REVIEW

**Comment by ShivamNegi** [comment]
The `CODE_OF_CONDUCT.md` needs to be added in `pages` of `make.jl` file.

**Comment by ShivamNegi** [comment]
Looks good to me. Lets just squash the commit and this is ready to go in.

**Comment by henrykironde** [comment]
Ok merge it in please. thanks

**Comment by ShivamNegi** [comment]
Just a sec. In the pages you have missed a `,` before `Code of conduct`.

File: docs/make.jl [hunk 1]
```diff
@@ -14,6 +14,7 @@ makedocs(
         "Installation Guide" => "tutorial.md",
         "Source" => "index.md",
         "Command Documentation" => "lib/public.md"
```
Comment (ShivamNegi) [inline docs/make.jl]: `,` missing.

File: docs/make.jl [hunk 2]
```diff
@@ -13,7 +13,7 @@ makedocs(
         "Home" => "intro.md",
         "Installation Guide" => "tutorial.md",
         "Source" => "index.md",
-        "Command Documentation" => "lib/public.md"
+        "Command Documentation" => "lib/public.md",
         "Code of conduct" => "CODE_OF_CONDUCT.md"
```
Comment (ShivamNegi) [inline docs/make.jl]: I would prefer the `C` capital in `Code of conduct` > `Code of Conduct`.

**Comment by henrykironde** [comment]
Okey docs are building for me

**Comment by ShivamNegi** [comment]
Thanks! Just squash the commits and I will merge it.

**Comment by henrykironde** [comment]
I have cleaned up some of the commits, 
But you could also use the arrow on the `merge pull request` green button to select `squash and merge`

**Pull request merged**